### DISCUSSION TITLE
[Feature] "Image Interogator"

### DISCUSSION THREAD

**Discussion by somebody-somewhere-over-the-rainbow** [immich-app/immich#2519] [Feature Request]
### The feature

Using something like stable diffusion to provide vastly improved tags for your images. The idea is born from a post in the photoprism repo that already has a script that could be build upon. See https://github.com/photoprism/photoprism/discussions/3419 for more details

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
We're already using CLIP to generate embeddings for search. We could look into using those embeddings to generate descriptions as well.

**Comment by alextran1502** [comment]
Pretty cool! Thank you for the headsup
